bin/test_unit

```diff
@@ -2,20 +2,17 @@
 
 set -eox pipefail
 
-rm -rf unit-test
-mkdir unit-test
-
-junit_output_file="unit-test/junit.output"
-
-rm -f "$junit_output_file"
-touch "$junit_output_file"
+junit_output_file="./junit.output"
 
 echo "Building unit test image..."
 docker build -f Dockerfile.test -t secrets-provider-for-k8s-test-runner:latest .
 
 echo "Running unit tests..."
 set +e
-  docker run --rm -t secrets-provider-for-k8s-test-runner:latest \
+  docker run --rm -t \
+             --volume "$PWD"/:/secrets-provider-for-k8s/test/ \
+             secrets-provider-for-k8s-test-runner:latest \
+             -coverprofile="./test/c.out" \
              ./cmd/... \
              ./pkg/... \
              | tee -a "$junit_output_file"
```

Comment:
what happens if we run this script twice? will it delete the output of the previous run?

Reply:
Yes. I verified it.(relevant for local test only)